lib/icalendar/deserialize.ex

```diff
@@ -8,10 +8,43 @@ defimpl ICalendar.Deserialize, for: BitString do
   alias ICalendar.Util.Deserialize
 
   def from_ics(ics) do
-    ics
-    |> String.trim()
-    |> String.split("\n")
-    |> Enum.map(&String.trim_trailing/1)
-    |> Deserialize.build_event()
+    events =
+      ics
+      |> String.trim()
+      |> String.split("\n")
+      |> Enum.map(&String.trim_trailing/1)
+      |> get_events()
+
+    # return single event if only one was found in ics file so the tests pass
```

Comment:
Can we make it always return a list please? Thank you

Reply:
This would be a breaking change. I guess upping the version to 1.0.0 will address that though.  I also updated the tests that fail as a result of this